### PR TITLE
feat: authorization for Renku 2.0 groups and namespaces

### PR DESCRIPTION
**Pull request by olevski**
/deploy

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -35,22 +35,29 @@ User-Facing Changes
...
 
+
 Internal Changes
 ~~~~~~~~~~~~~~~~
 
 **🌟 New Features**
 
 - **Data Services**: Updated the authorization for access to Renku 2.0 projects to work with Authzed DB,
   a 3rd party database dedicated to saving authorization data and making authorization decisions
+- **Data Services**: Updated the authorization for access to Renku 2.0 namespaces and groups with Authzed DB
```
Comment (lokijuhy) [inline CHANGELOG.rst:46]: Looks like something funny is going on here?

Comment (lokijuhy) [reply CHANGELOG.rst:46]: Ooh was this intentional? It looks like a typo. Can we please combine these into one line?
Also, what do you mean by authorization to namespaces? What does that mean distinct from groups? I would expect it to say "...authorization for access to Renku 2.0 projects and groups..."

File: CHANGELOG.rst [hunk 2]
```diff
@@ -35,22 +35,29 @@ User-Facing Changes
...
 - **Search Services**: Support processing v2 schema messages (alongside with v1). Making the query parser
   more lenient to not raise parsing errors.
 
+**🐞 Bug Fixes**
+
+- **Data Services**: allow removing tolerations and affinities on resource pool via PATCH requests
```
Comment (lokijuhy) [inline CHANGELOG.rst:52]:
```suggestion
- **Data Services**: Allow removing tolerations and affinities on resource pools via PATCH requests
```

**Review by lokijuhy — CHANGES_REQUESTED** [review]

**Comment by olevski** [comment]
@lokijuhy please note that this PR needs also a ui version before it can be complete and merged. This should happen today or Monday. But it will most likely be on Monday.

**Review by Panaetius — APPROVED** [review]

**Review by lokijuhy — APPROVED** [review]

**Pull request merged**